lib/cosmos/gui/utilities/script_module_gui.rb

```diff
@@ -42,7 +46,7 @@ def ask_string(question, allow_blank = false)
         break if allow_blank or (not answer.nil? and answer.strip.length != 0)
       end
 
-      Logger.info "User entered '#{answer}' for '#{question}'"
+      Logger.info "User entered '#{answer}' for '#{question}'" unless password
```

Comment:
Shouldn't we still log something when this dialog comes up. Maybe ```"User entered '#{'*' * password.length}' for '#{question}'"``` if they use the password option.